Orange/widgets/model/owrandomforest.py

```diff
@@ -27,7 +27,8 @@ class OWRandomForest(OWBaseLearner):
     index_output = settings.Setting(0)
 
     class Error(OWBaseLearner.Error):
-        not_enough_features = Msg("Insufficient number of attributes ({})")
+        not_enough_features = Msg(
+            'Number of considered attributes is too high (max {})!')
```

Comment:
I looked at the message and found it strange so I tried to change it to something a bit more informative. I tried your message, but sadly it's just a bit too long. What about `"Considering more attributes than available"` or `"Considering more attributes than total number of attributes"`?